evol/population.py

```diff
@@ -188,24 +192,26 @@ def breed(self, parent_picker, combiner, population_size=None, **kwargs) -> 'Pop
             Arguments are only passed to the functions if they accept them.
         :return: self
         """
-        parent_picker = select_arguments(parent_picker)
+        # TODO I'm not exactly sure, on an API level, where the `ensure_list` and `select_arguments`
+        # should go, if here on inside offspring_generator
+        parent_picker = ensure_list(select_arguments(parent_picker))
```

Comment:
The parent pick is also allowed to return just 1 individual. In fact, technically, we even allow for no parent to be passed by the parent picker.

Reply:
I'm indeed not sure if we want to enforce the return type of the parent picker. I can live with not allowing a single individual but enforcing that to be a list (tuple?) with a single element. Can we think of use cases to return something else than an ordered collection of individuals? Something with weights?